src/id.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * This module provides functionality for generating random identifiers.
+ * Studies can use these identifiers to uniquely label events and other
+ * items of interest.
+ * @module webScience.id
+ */
+
+/**
+* Generate an ID, a random 128-bit value represented as a hexadecimal string.
+* @returns {string} The new random ID.
+*/
+export function generateId() {
```

Comment:
This is probably OK but we should consider using [uuidjs](https://github.com/uuidjs/uuid) just because making sure it's RFC compliant is annoying. Using `window.crypto` is good though, it might be fine to keep it this way.

Reply:
I've been waffling on whether to use UUIDs or something else (see https://github.com/mozilla-rally/web-science/issues/19). UUIDs are antiquated and inefficient... but they are common. 🤷‍♂️

Reply:
> UUIDs are antiquated and inefficient... but they are common. 🤷‍♂️

Yeah a lot of that analysis tooling expects them :/ it's frustrating.